model.py

```diff
@@ -38,16 +38,14 @@ class IfStatement(BaseModel, Generic[T]):
 ###################
 
 
-class BasePackage(StrictBaseModel):
+class SimplePackage(StrictBaseModel):
     name: str = Field(description="The package name")
-
-
-class SimplePackage(BasePackage):
     version: str = Field(description="The package version")
 
 
-class ComplexPackage(BasePackage):
-    pass
+class ComplexPackage(StrictBaseModel):
+    name: Optional[str] = Field(None, description="The package name, this can be overwritten per output")
```

Comment:
Actually, I think we should say that this is ignored (only used e.g. for the feedstock name or to otherwise identify the main recipe outptu for humans). wdyt?

Reply:
Should we maybe rename it then? `recipe-name`? Or perhaps call the top-level not package but `recipe` in that case?

```json
recipe:
   name: bla
   version: bla
```

Reply:
I changed it, let me know what you think.